### ISSUE TITLE
No indentation after `if` statement

### ISSUE THREAD

**Issue by Phrogz** [trixnz/vscode-lua#64]
Typing the following code with newlines only produces the indentation seen:

```lua
function foo()
   if true then
   a
else
   b
end
end
```

Indent after `function` open is working, indent after `else` is working, de-indent upon `end` is working, but indent after `if/then` is not working. Side-effect is that taking file with proper indentation like:

```lua
function foo()
   c
   if true then
      a
      b
   else      
      d
   end
end
```
and then using alt+downarrow to move line C into `else` produces this result:
```lua
function foo()
   if true then
   a
   b
else      
   c
      d
   end
end
```

Seems a regression based on what #19 describes.

Extension Info:
```
Name: vscode-lua
Id: trixnz.vscode-lua
Description: Intellisense and Linting for Lua
Version: 0.12.4
Publisher: trixnz
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=trixnz.vscode-lua
```
Extension is using 5.3 mode, in case that matters.

VS Code:
```
Version 1.31.1 (1.31.1)
1b8e8302e405050205e69b59abb3559592bb9e60
2019-02-12T02:10:22.500Z
```

**Comment by Nexela** [comment]
This is a bug in the official vs code repo I believe

**Comment by Phrogz** [comment]
I agree. It is fixed in Version 1.32.0-insider (1.32.0-insider). Sorry for the noise.
